sonar-db/src/main/java/org/sonar/db/user/GroupDao.java

```diff
@@ -60,7 +58,7 @@ public GroupDto selectByName(DbSession session, String key) {
   }
 
   public List<GroupDto> selectByNames(DbSession session, Collection<String> names) {
-    return executeLargeInputs(names, new SelectByNames(mapper(session)));
+    return executeLargeInputs(names, partitionOfNames -> mapper(session).selectByNames(partitionOfNames));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)